src/tiled/propertybrowser.cpp

```diff
@@ -857,6 +835,55 @@ QtVariantProperty *PropertyBrowser::createProperty(PropertyId id, int type,
     return property;
 }
 
+void PropertyBrowser::addNewProperties(Object* object) {
```

Comment:
Please keep in mind the coding style used in this project, which places the opening `{` for functions on the next line.